distributed/run_packer.sh

```diff
@@ -0,0 +1,15 @@
+#! /bin/bash
+set -eux
+
+export AWS_PROFILE="cmr-sit"
+
+ownerID=$(python3 query_AWS_for_packer.py ami)
+vpcID=$(python3 query_AWS_for_packer.py vpc)
```

Comment:
Is the EC2 instance running in the CMR vpc and subnets or somewhere else?  If so, using terraform makes this a lot easier and you won't need all this software.  If not then please ignore my comment.

Reply:
It is indeed running on CMR VPC and one of its private subnets. I'm hoping that next PI I'll get to develop this proof of concept into an actual service, at which point I think Terraform would make much more sense than having to query boto3 so many times. I don't have any experience with Terraform, yet, so I used boto3 to help get the infra bootstrapped to test the DuckDB deployment options.